hubblestack/utils/decorators/jinja.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Jinja-specific decorators
+"""
+# Import Python libs
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class JinjaFilter(object):
```

Comment:
We actually do not need Jinja thing. It's for Jinja template support.
Its used in Salt for some other purpose when Master/Minion is there. 
https://docs.saltstack.com/en/master/topics/jinja/index.html#understanding-jinja

We can safely remove the usage of JinjaFilter

Reply:
Now that yoou mention it, I remeber we talked about this and decided it wasn't needed. I'll remove it right away